WIP-CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Used FQDN for services connecting from the workload cluster to the service cluster to prevent resolve timeouts
 
 ### Added
+-  Network policies for `coredns` and `dnsAutoscaler`.
```

Comment:
Nit: Move it either one line down or append it to the list.

Reply:
done